internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-nlb-prod.stack.yml

```diff
@@ -619,8 +619,8 @@ Resources: # If a bucket URL is specified, that means the template exists.
     Condition: HasAssociatedDomain
     Properties:
       ServiceToken: !GetAtt NLBCustomDomainFunction.Arn
-      LoadBalancerHostedZoneID: !GetAtt PublicNetworkLoadBalancer.CanonicalHostedZoneID
-      LoadBalancerDNS: !GetAtt PublicNetworkLoadBalancer.DNSName
+      PublicAccessHostedZoneID: !GetAtt PublicNetworkLoadBalancer.CanonicalHostedZoneID
+      PublicAccessDNS: !GetAtt PublicNetworkLoadBalancer.DNSName
```

Comment:
have you verified this works with an _existing_ nlb running the custom domain function without any other changes? (i.e., aliases stay the same)

Reply:
yeah it's all good